dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md

```diff
@@ -1,35 +1,119 @@
+```
+
+**Step 3:** We embed these unique ONS job titles and the input job title using the `all-MiniLM-L6-v2` Sentence Tranformers pretrained model.
+
+**Step 4:** We then calculate the cosine similarity between the embedded input job title and all the embedded ONS job titles.
+
+In our example, the cosine similarity scores for each input job title (row) and each ONS SOC data job title (columns) are:
+
+|                              | data scientist | data analyst computing | data analyst | analyst developer | meter assembler | motor assembler electric | motor assembler engineering |
+| ---------------------------- | -------------- | ---------------------- | ------------ | ----------------- | --------------- | ------------------------ | --------------------------- |
+| Data Scientist               | **1.0**        | 0.69                   | 0.81         | 0.56              | 0.20            | 0.07                     | 0.15                        |
+| Electric motor assembler     | 0.07           | 0.20                   | 0.11         | 0.15              | 0.52            | **0.81**                 | 0.80                        |
+| Data visualisation developer | 0.53           | 0.57                   | 0.59         | 0.47              | 0.22            | 0.04                     | 0.17                        |
+
+**Step 5:** Finally, we find the SOC information for the ONS job title with the highest similarity as long as it is over a certain threshold (default is 0.67). If there is no single job title with a particularly high similiarity, then we use a consensus approach at the SOC 2020 4-digit level (default to having over 3 matches with over 0.5 similarity score).
+
+With the default values, the final matches for each inputted job title would be:
+
+|                              | ONS job title matched to | SOC     | SOC description                                 |
+| ---------------------------- | ------------------------ | ------- | ----------------------------------------------- |
+| Data Scientist               | data scientist           | 2433/02 | Data scientists                                 |
+| Electric motor assembler     | motor assembler electric | 8141/00 | Assemblers (electrical and electronic products) |
+| Data visualisation developer | None                     | None    | None                                            |
+
+However, if we had set slightly different conditions for the consensus approach, another outcome could be that the "Data visualisation developer" job title was mapped to the SOC "2134 - Programmers and software professionals" since 2 out of the 4 matches with over 0.45 similarity were from this 4-digit SOC.
+
+# Occupational Green Measures
```

Comment:
not here but i would add an additional line under usage to say that thats to map to green measure